pkg/dockerfile/generator.go

```diff
@@ -247,10 +248,7 @@ func (g *Generator) installCython() string {
 	return "RUN --mount=type=cache,target=/root/.cache/pip pip install cython==\"0.29.34\""
 }
 func (g *Generator) installSieve() string {
-	sieveExternal := "sievedata-0.0.1.1-py3-none-any.whl"
-	format := "COPY %s /tmp/%s\n RUN --mount=type=cache,target=/root/.cache/pip pip install /tmp/%s"
-	line2 := fmt.Sprintf(format, sieveExternal, sieveExternal, sieveExternal)
-	return fmt.Sprintf("%s", line2)
+	return "RUN --mount=type=cache,target=/root/.cache/pip pip install sievedata"
```

Comment:
i think this might need to installed from the local wheel instead of from pypi?

how will this work when we're devving and we change `SIEVE_WHEEL_NAME` in circleci?